.github/workflows/github_ci_logs.yml

```diff
@@ -0,0 +1,32 @@
+name: Github CI log archiver
+on:
+  workflow_run:
+    workflows: ["Github CI"]
+    types:
+      - completed
+
+jobs:
+  save_logs:
+    name: Save run logs
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download log archive
+      run: |
+        curl -sI -u ${{ github.repository_owner }}:${{ github.token }} \
+          https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.event.workflow_run.id }}/logs > response.txt
+        DOWNLOAD_LINK="https:"$(cat response.txt | grep -Fi Location: | cut -d: -f3)
```

Comment:
Bash style nit: I'd prefer `"https:$(cat ...)"` rather than `"https:"$(cat ...)`. I think they end up meaning exactly the same thing, even in the presence of special characters; but the latter feels saner and clearer to me that the _whole thing_ is supposed to be a string.